src/components/Articles.jsx

```diff
@@ -26,6 +36,29 @@ const Articles = ({ setIsLoading, scrollUp }) => {
         })
       }, [article_id])
       //
+
+      const [username, setUsername] = useState('');
+      const [body, setBody] = useState('');
+
+      const handleSubmit = (event) => {
+        event.preventDefault()
+        
+        if (userNames.includes(username)) {
```

Comment:
this works great! it would be good if there was a message to the user to tell them why their comment isn't posting here